src/CMakeLists.txt

```diff
@@ -1,4 +1,5 @@
 add_library(thruster SHARED thruster.cpp)
+add_dependencies(thruster ${${PROJECT_NAME}_EXPORTED_TARGETS} ${catkin_EXPORTED_TARGETS})
```

Comment:
what does this do? Why does thruster depend on anything? Is this ensuring that Gazebo dependencies are met before thruster is built?